handler/pullRequestHandler.go

```diff
@@ -101,6 +86,57 @@ That's something we need before your Pull Request can be merged. Please see our
 	}
 }
 
+// VerifyPullRequestDescription checks that the PR has anything in the body.
+// If there is no body, an 'invalid' label is added and comment posted to the PR with a link to the contributing guide.
+func VerifyPullRequestDescription(req types.PullRequestOuter, contributingURL string, config config.Config) {
+	ctx := context.Background()
+	token := getAccessToken(config, req.Installation.ID)
+	client := factory.MakeClient(ctx, token, config)
+
+	if req.Action == "opened" {
+		if missingDescription(req.PullRequest) {
+			fmt.Println("Applying invalid label")
```

Comment:
What if we extracted a const for the invalid label so we could change it in the future potentially?

For instance:

```
const noDescriptionLabel = "invalid"
```

Reply:
good idea! updated to a const at the top of the file